src/main/scala/diplomaticobjectmodel/DiplomaticObjectModelUtils.scala

```diff
@@ -168,7 +168,7 @@ object DiplomaticObjectModelAddressing {
 
   def getOMMemoryRegions(name: String, resourceBindings: ResourceBindings, omRegMap: Option[OMRegisterMap] = None): Seq[OMMemoryRegion]= {
     resourceBindings.map.collect {
-      case (x: String, seq: Seq[Binding]) if (DiplomacyUtils.regFilter(x)) =>
+      case (x: String, seq: Seq[Binding]) if (DiplomacyUtils.regFilter(x)) || (DiplomacyUtils.rangeFilter(x))=>
```

Comment:
Nit: Put a space before the `=>` arrow.
Nit 2: I think the parentheses are unnecessary. Does the following suggestion compile?

```suggestion
      case (x: String, seq: Seq[Binding]) if DiplomacyUtils.regFilter(x) || DiplomacyUtils.rangeFilter(x) =>
```

Reply:
Does the gating expression in getOMPortMemoryRegions need the same set of conditions?

Reply:
Oh, sorry, I didn't read this PR carefully enough and mixed it up with the other one. This change should *only* go on the ports, although it's probably not too harmful to put it on the memories.

Reply:
Added